src/commands/deploy.js

```diff
@@ -80,13 +80,13 @@ class DeployCommand extends Command {
     }
 
     if (!deployFolder) {
-      this.log('Please provide a deploy path relative to:')
+      this.log('Please provide a publish directory (e.g. "public" or "dist" or "."):')
       this.log(process.cwd())
       const { promptPath } = await inquirer.prompt([
         {
           type: 'input',
           name: 'promptPath',
-          message: 'deploy path',
+          message: 'Publish Directory',
```

Comment:
```suggestion
          message: 'Publish directory',
```

Minor change, and I'm sure it's not consistent throughout the CLI yet, but we're standardizing on sentence case across the board, so might as well make this particular prompt match before we do a broader update to all prompts.